package.json

```diff
@@ -4,12 +4,15 @@
   "private": true,
   "homepage": "http://coding-coach.github.io/",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.4",
+    "@fortawesome/free-brands-svg-icons": "^5.3.1",
```

Comment:
These three dependencies are adding about ~170kb to the final bundle (Compressed and gziped), is this something we really need? Can we just wrap the SVGs we need?

Reply:
@crysfel Which tool did you use to analyse the bundle size? While building, I am getting this - 67.12 KB (-251 B)  build/static/js/main.7f3fc8f7.js

Reply:
@crysfel I analyzed the bundle size with source-map-explorer and it showed the ~29kb of size (compressed).

Reply:
I only searched the libraries in https://bundlephobia.com, I will build it later and see the actual size in our production bundle.